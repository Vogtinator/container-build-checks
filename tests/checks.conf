[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
