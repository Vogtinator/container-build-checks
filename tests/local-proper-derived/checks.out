Looking at local-proper-derived.containerinfo
Hint: No release and disturl found in containerinfo, probably a local osc build. Further analysis might be misleading.
Tag: opensuse/example:latest
Tag: opensuse/example:5.7
Tag: opensuse/example:5.7.7.21
Tag: kubic/testtag:latest
Detected image specific label prefix: org.opensuse.example
Hint: No disturl in containerinfo, local build?

container-build-checks done. Hints: 2 Warnings: 0 Errors: 0
Exited with 0
